packages/cells/src/widget.ts

```diff
@@ -211,6 +211,17 @@ class Cell extends Widget {
         this.editor.setOption(key, options.editorConfig[key]);
       });
     }
+
+  }
+
+  /**
+   * Modify some state for initialization.
+   *
+   * Should be called at the end of the subclasses's constructor.
+   */
+  protected initializeState() {
+    const jupyter = this.model.metadata.get('jupyter') || {} as any;
+    this.inputHidden = jupyter.source_hidden || false;
```

Comment:
I think a check for `jupyter.source_hidden === true` would be more robust, since this would also pass for truthy values like `"false"`.

Reply:
That makes a lot of sense, I hadn't thought of malformed cell metadata.